src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -1991,6 +2023,7 @@ public EntryResult tManufacturedProduct2Medication(Consumable cdaConsumable, IBu
 	public EntryResult tMedicationActivity2MedicationStatement(MedicationActivity cdaMedicationActivity,
 			IBundleInfo bundleInfo) {
 
+		LocalBundleInfo localBundleInfo = new LocalBundleInfo(bundleInfo);
```

Comment:
LocalBundleInfo is necessary to capture the updated medication code map, as we don' want to edit the parameters of the function.